widget/scroller.go

```diff
@@ -8,9 +8,126 @@ import (
 	"fyne.io/fyne/theme"
 )
 
+type scrollBarRenderer struct {
+	scrollBar *scrollBar
+	bar       *canvas.Rectangle
+
+	objects []fyne.CanvasObject
+}
+
+func (s *scrollBarRenderer) Layout(size fyne.Size) {
+	// Positions the bar on the far right of the container
+	s.scrollBar.Move(fyne.NewPos(size.Width-theme.ScrollBarSize(), 0))
```

Comment:
I think this should be in the ScrollContainer renderer as it's our position within parent (noted the insertion point below)